tests/expose_test.go

```diff
@@ -179,15 +311,30 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				Expect(nodes.Items).ToNot(BeEmpty())
 				for _, node := range nodes.Items {
 					Expect(node.Status.Addresses).ToNot(BeEmpty())
-					nodeIP := node.Status.Addresses[0].Address
+
+					nodeIP, err := getNodeInternalIPByFamily(node, ipFamily)
+					if err != nil {
```

Comment:
Please don't assume the err is not find the ipv6 address. In case of ipv4 add regular error handling (assertion). In case of ipv6 do the workaround.

Reply:
Removed the commit that introduced this.